protected/humhub/modules/user/views/must-change-password/index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use humhub\modules\user\models\Password;
+use humhub\widgets\Button;
+use yii\helpers\Url;
+use yii\helpers\Html;
+use yii\bootstrap\ActiveForm;
+use humhub\widgets\SiteLogo;
+
+/* @var $model Password */
+
+$this->pageTitle = Yii::t('UserModule.auth', 'Change password');
+?>
+<div class="container" style="text-align: center;">
+    <?= SiteLogo::widget(['place' => 'login']); ?>
+    <br>
+
+    <div class="row">
+        <div id="must-change-password-form" class="panel panel-default animated bounceIn"
+             style="max-width: 300px; margin: 0 auto 20px; text-align: left;">
+            <div class="panel-heading"><?= Yii::t('UserModule.auth', '<strong>Change</strong> password'); ?></div>
+            <div class="panel-body">
+
+                <?php $form = ActiveForm::begin(); ?>
+
+                <p><?= Yii::t('UserModule.auth', 'Please change your password.'); ?></p>
+
+                <?php echo $form->field($model, 'currentPassword')->passwordInput(['maxlength' => 45]); ?>
```

Comment:
Please rather use `<?=`

Reply:
ah yes, I just copied this from another file, maybe this https://github.com/humhub/humhub/blob/enh/3996-force-password-change/protected/humhub/modules/user/views/account/changePassword.php#L13.
Fixed in commit https://github.com/humhub/humhub/commit/fc918208d1605da4b39bc739200502ae3f48c43f.